### PR TITLE
#729: Add rules_jvm_external to Bazel and add a test maven dependency

### PR DESCRIPTION
**Pull request by tkilias**
Part of https://github.com/exasol/script-languages-release/issues/729

### CODE REVIEW

File: exaudfclient/base/javacontainer/BUILD [hunk 1]
```diff
@@ -134,27 +134,11 @@ java_library(
...
-            ":ExaConnectionInformation.java",
-            ":ExaDataTypeException.java",
-            ":ExaImportSpecification.java",
-            ":ExaExportSpecification.java",
-            ":ExaIterationException.java",
-            ":ExaIterator.java",
-            ":ExaMetadata.java",
-            ":ExaUDFException.java"
-            ],
-    resources = ["//:LICENSE-exasol-script-api.txt"],
```
Comment (kaklakariada) [inline exaudfclient/base/javacontainer/BUILD:154]: Do you also need to delete this file?

Comment (tomuben) [reply exaudfclient/base/javacontainer/BUILD:154]: I think you are right.  I deleted it.

File: exaudfclient/base/WORKSPACE [hunk 2]
```diff
@@ -9,6 +9,36 @@ http_archive(
     url = "https://github.com/bazelbuild/bazel-skylib/releases/download/{}/bazel-skylib.{}.tar.gz".format(skylib_version, skylib_version),
 )
 
+RULES_JVM_EXTERNAL_TAG = "4.2"
```
Comment (tkilias) [inline exaudfclient/base/WORKSPACE:12]: maybe we should update, or we do this in a extra PR

https://github.com/bazelbuild/rules_jvm_external/releases/tag/6.2

**Review by tkilias — COMMENTED** [review]
approved, but see my comment. 
PS: I can't approve the PR officially, because I am the creator of it

File: exaudfclient/base/WORKSPACE (hunk 2 above)
Comment (tomuben) [reply exaudfclient/base/WORKSPACE:12]: Tried it, but it does not work. 
Anyway, will be updated in https://github.com/exasol/script-languages/pull/439

**Review by tomuben — APPROVED** [review]

**Pull request merged**